app/src/main/java/com/ajdi/yassin/popularmovies/LoadingActivity.java

```diff
@@ -0,0 +1,29 @@
+package com.ajdi.yassin.popularmovies;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.os.Handler;
+import android.widget.ImageView;
+
+
+import com.bumptech.glide.Glide;
+
+public class LoadingActivity extends Activity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_loading);
+        ImageView imageView=(ImageView)findViewById(R.id.image_loading);
+        Glide.with(getBaseContext()).load(R.raw.marvle).into(imageView);
```

Comment:
Is the marvel gif free and open source?

Reply:
Marvel is said to be fine when the logo comes out. So I'll change it to another gif.

Reply:
if it's open-source then no need to change it.